README.md

```diff
@@ -84,6 +84,7 @@ and a transform stream to decompress the file.
 | [options.check] | <code>Boolean</code> | <code>false</code> | enable write check |
```

Comment:
If `check` is true, and checksumAlgorithms is missing / empty, should we ensure that at least one 'default' algorithm gets calculated? (or does it do that already?)